ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/reading/LedgerStateOperationsReader.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.validator
+
+import com.daml.ledger.validator.LedgerStateOperations.{Key, Value}
+import com.daml.ledger.validator.reading.StateReader
+
+import scala.concurrent.{ExecutionContext, Future}
+
+final class LedgerStateOperationsReader[LogResult](operations: LedgerStateOperations[LogResult])
```

Comment:
This name suggests you're reading from `LedgerStateOperations` which is confusing. Maybe call it `LedgerStateOperationsStateReaderAdaptor` or `LedgerStateOperationsReaderAdaptor` instead?